content/sensu-go/5.16/sensuctl/reference.md

```diff
@@ -886,18 +886,33 @@ To install a sensuctl command from Bonsai or a URL:
 sensuctl command install [ALIAS] ([NAMESPACE/NAME]:[VERSION] | --url [ARCHIVE_URL] --checksum [ARCHIVE_CHECKSUM]) [flags]
 {{< /highlight >}}
 
+You can install a command by using the Bonsai asset name or from a URL.
+
 **To install a command using the Bonsai asset name**, replace `[NAMESPACE/NAME]` with the name of the asset from Bonsai.
 `[:VERSION]` is only required if you require a specific version or are pinning to a specific version.
 If you do not specify a version, sensuctl will fetch the latest version from Bonsai.
 
-For example, to install a command from the [Sensu Go Email Handler Plugin][37] with no flags:
+Replace `[ALIAS]` with a unique name for the command.
+For example, for the [Sensu EC2 Discovery Plugin][37], you might use the alias `sensu-ec2-discovery`. 
+`[ALIAS]` is required.
+
+Replace `[flags]` with the flags you want to use.
+Run `sensuctl command install -h` to view flags.
+Flags are optional and apply only to the `install` command &mdash; they are not saved as part of the command you are installing.
+
+To install a command from the [Sensu EC2 Discovery Plugin][37] with no flags:
 
 {{< highlight shell >}}
-sensuctl command install sensu-email-handler (sensu/sensu-email-handler:0.2.0)
+sensuctl command install sensu-ec2-discovery (portertech/sensu-ec2-discovery:0.3.0)
```

Comment:
```suggestion
sensuctl command install sensu-ec2-discovery portertech/sensu-ec2-discovery:0.3.0
```

I think the parentheses in the `sensuctl command install` usage output are meant to indicate that one should provide either a `[NAMESPACE/NAME]:[VERSION]` or a `url` and `checksum`, meaning those parentheses should not be used in the actual command.

That said, this command isn't yet working for me, with or without the parentheses. I'm getting `Error: namespace cannot be empty` but I'm not sure if this a problem with sensuctl or what bonsai is returning.